MvvmCross/Binding/iOS/Target/MvxUITextViewTextTargetBinding.cs

```diff
@@ -46,7 +46,16 @@ public override void SubscribeToEvents()
                 return;
             }
 
-            target.Changed += this.EditTextOnChanged;
+			var textStorage = target.LayoutManager?.TextStorage;
+
+			if (textStorage == null)
+			{ 
+			    MvxBindingTrace.Trace(MvxTraceLevel.Error,
+						  "Error - NSTextStorage of UITextView is null in MvxUITextViewTextTargetBinding");
+				return;
+			}
+
+            textStorage.DidProcessEditing += this.EditTextOnChanged;
```

Comment:
The way I read the [documentation](https://developer.apple.com/reference/uikit/nstextstoragedelegate) (although I am also a bit confused) is that the methods listed under "Handling Text-Edited Messages" are the currently supported methods and replace the methods listed under the "Deprecated" header.

Also the Xamarin [documentation](https://developer.xamarin.com/api/type/MonoTouch.UIKit.NSTextStorage/) doesn't mention anything about the events being deprecated.